src/elfpy/pricing_models.py

```diff
@@ -472,14 +475,14 @@ def open_short(self, trade_details):
             will be conditional on the pricing model
         """
         trade_results = self.calc_out_given_in(
-            trade_details["trade_amount"],
-            trade_details["share_reserves"],
-            trade_details["bond_reserves"],
-            trade_details["token_out"],
-            trade_details["fee_percent"],
-            trade_details["stretched_time_remaining"],
-            trade_details["init_share_price"],
-            trade_details["share_price"],
+            trade_details.trade_amount,
```

Comment:
Aren't we passing in a `user_action`? Can we go ahead and either rename that here or in `swap`?